lib/experiments-page.php

```diff
@@ -89,18 +89,6 @@ function gutenberg_initialize_experiments_settings() {
 		)
 	);
 
-	add_settings_field(
-		'gutenberg-theme-previews',
-		__( 'Block Theme Previews', 'gutenberg' ),
-		'gutenberg_display_experiment_field',
-		'gutenberg-experiments',
-		'gutenberg_experiments_section',
-		array(
-			'label' => __( 'Enable Block Theme Previews', 'gutenberg' ),
-			'id'    => 'gutenberg-theme-previews',
-		)
-	);
-
```

Comment:
I would be better to keep this PR surface area small and not remove the experiment in this PR. 

Why? Because if we decide we need to roll back to making this an experiment we'll also have to revert all the other code changes. If there have been other changes to those files since then, the unfortunate developer will have to handle that which isn't something we should impose upon them.

Let's make the removal of the experiment a dedicated follow up PR.

Reply:
I'm going to create another PR for https://github.com/WordPress/gutenberg/commit/1f8961aa2b89552fc6f6b6e521dc696923c85547 👍 
c.f. https://github.com/WordPress/gutenberg/pull/50863#discussion_r1204925282